main.go

```diff
@@ -190,43 +212,56 @@ func main() {
 	}
 	filePath := os.Args[1]
 	debug("## Loading " + filePath)
+	
+	var test Test
+	var summary Summary
+	var subsetPartition map[int][]int
+
+	readTestFile(filePath, &test)
```

Comment:
`test := readTestFile()`

Since Go functions allow to return stuff, let's use.